test/integration/binaries/test_binaries.py

```diff
@@ -0,0 +1,38 @@
+import os
+import shutil
+import subprocess
+
+import pytest
+
+from utils import config_name
+
+
+@pytest.mark.slow
+def test_binaries(tt_cmd, tmpdir):
+    # Copy the test bin_dir to the "run" directory.
+    test_app_path = os.path.join(os.path.dirname(__file__), "bin")
+    shutil.copytree(test_app_path, tmpdir + "/bin")
+
+    configPath = os.path.join(tmpdir, config_name)
+    # Create test config
+    with open(configPath, 'w') as f:
```

Comment:
Add tests for:
- Empty bin_dir.
- Missing bin-dir.

Reply:
Added